blur/tests/blur-exchange-utils.ts

```diff
@@ -0,0 +1,297 @@
+import { newMockEvent } from "matchstick-as"
+import { ethereum, Address, BigInt, Bytes } from "@graphprotocol/graph-ts"
+import {
+  AdminChanged,
+  BeaconUpgraded,
+  Closed,
+  Initialized,
+  NewBlockRange,
+  NewExecutionDelegate,
+  NewFeeRate,
+  NewFeeRecipient,
+  NewGovernor,
+  NewOracle,
+  NewPolicyManager,
+  NonceIncremented,
+  Opened,
+  OrderCancelled,
+  OrdersMatched,
+  OwnershipTransferred,
+  Upgraded
+} from "../generated/BlurExchange/BlurExchange"
+
+export function createAdminChangedEvent(
```

Comment:
all funcs in this file seem to not be used, please remove whatever is not needed

Reply:
done